utils.go

```diff
@@ -288,15 +288,15 @@ func (s *Service) detectLocation(name string) (location string, err error) {
 	defer func() {
 		err = s.formatError("detect_location", err, "")
 	}()
-
-	url := fmt.Sprintf("%s://%s.%s:%d", s.config.Protocol, name, s.config.Host, s.config.Port)
+	
+	url := fmt.Sprintf("%s://%s/%s", s.config.Protocol, "qingstor.com" , name )
```

Comment:
Please use `s.config.Hos` instead of `qingstor.com`.

Reply:
Please also take `Port` into consideration.

Reply:
done